src/command_modules/azure-cli-policyinsights/azure/cli/command_modules/policyinsights/_params.py

```diff
@@ -0,0 +1,113 @@
+# --------------------------------------------------------------------------------------------
+# Copyright (c) Microsoft Corporation. All rights reserved.
+# Licensed under the MIT License. See License.txt in the project root for license information.
+# --------------------------------------------------------------------------------------------
+
+from azure.cli.core.commands.parameters import resource_group_name_type
+
+from azure.cli.command_modules.resource._completers import (
+    get_policy_set_completion_list, get_policy_completion_list,
+    get_policy_assignment_completion_list, get_providers_completion_list, get_resource_types_completion_list)
+
+from ._validators import validate_resource
+
+
+def load_arguments(self, _):
+    for scope in ['state', 'event']:
+        with self.argument_context('policy {}'.format(scope)) as c:
+            c.argument(
+                'management_group_name',
+                options_list=['--management-group', '-m'],
+                arg_group='Scope',
+                help='Name of management group.')
+            c.argument(
+                'resource_group_name',
+                arg_type=resource_group_name_type,
+                arg_group='Scope')
+            c.argument(
+                'resource',
+                options_list=['--resource'],
+                validator=validate_resource,
+                arg_group='Resource ID',
+                help='Resource ID or resource name.')
```

Comment:
I'd recommend adding an extra line to the help along the lines of, 
`If an name is given, please provide the resource_group and other relevant resource id arguments.`

This is because `--resource-group` will be under a different argument group.

Reply:
will do